auth/template.yaml

```diff
@@ -411,6 +411,9 @@ Resources:
         - onelogin
       AttributeMapping:
         email: email
+        email_verified: email_verified
+        phone_number: phone_number
+        phone_number_verified: phone_number_verified
```

Comment:
Q - are we now needing phone number? I noticed this [thread](https://gds.slack.com/archives/C08D5NG9K41/p1740740964111129) - from my understanding it's an optional attribute we currently don't have to store

Reply:
Ah, I didn't see this, I just noticed it was used in the PoC, will revert, cheers.

Reply:
Sorted